### DISCUSSION TITLE
No response from GitHub Support

### DISCUSSION THREAD

**Discussion by abhishekHegde2000** [community/community#83619] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have raised 2 new tickets from GitHub support, it been already a week, when can i expect reply from them??

**Comment by abhishekHegde2000** [comment]
@stevepopovich
Can you please assist me with my ticket: #2503835

**Comment by asmawi117** [comment]
any update from github support ?
mine hasn't got any response after 35 days
